overridable/doc.go

```diff
@@ -0,0 +1,3 @@
+// Package overridable provides data types representing values in campaign
+// specs that can be overridden for specific repositories.
+package overridable
```

Comment:
Do we need this file? Because the comment is duplicated in `overridable/overridable.go`?